examples/hata_basic_queue/bot.py

```diff
@@ -172,27 +181,59 @@ async def skip(ctx):
 @bot.commands
 @checks.guild_only()
 async def pause(ctx):
-    """Pause the current song."""
+    """Pauses the current song."""
     await ctx.client.data.lavalink.pause(ctx.guild.id)
     await ctx.reply("Paused player")
 
 
 @bot.commands
 @checks.guild_only()
 async def resume(ctx):
-    """Resume playing the current song."""
+    """Resumes playing the current song."""
     await ctx.client.data.lavalink.resume(ctx.guild.id)
     await ctx.reply("Resumed player")
 
 
+@bot.commands(aliases=['np'])
```

Comment:
Have you tested this? The commands() decorator being undocumented makes me not trust this.

Reply:
Tested and working :)